### ISSUE TITLE
Does it support volcano-scheduler?

### ISSUE THREAD

**Issue by ESonata** [Project-HAMi/HAMi#392]
If it supports the use of volcano-scheduler, is there a user manual available? Thank you.

**Comment by lengrongfu** [comment]
Yes, place refer to this project https://github.com/Project-HAMi/volcano-vgpu-device-plugin

**Comment by ESonata** [comment]
> Yes, please refer to this project https://github.com/Project-HAMi/volcano-vgpu-device-plugin

Thank you. If using volcano scheduler, can we also use scheduling strategies such as gang based on volcano.sh/vgpu number and volcano.sh/vgpu memory？

**Comment by ESonata** [comment]
@lengrongfu  Thank you. If using volcano scheduler, can we also use scheduling strategies such as gang based on volcano.sh/vgpu number and volcano.sh/vgpu memory？

**Comment by archlitchi** [comment]
> @lengrongfu Thank you. If using volcano scheduler, can we also use scheduling strategies such as gang based on volcano.sh/vgpu number and volcano.sh/vgpu memory？

yes

**Comment by MouseSun846** [comment]
Can the Volcano scheduler be used to schedule non - GPU virtualization capabilities, such as NPU and DCU? Essentially, we don't want to introduce multiple schedulers into the cluster.

**Comment by mesutoezdil** [comment]
Hi,

We are doing a cleanup of open issues to keep the backlog manageable. This issue was opened before 2026 and has not seen recent activity.

Please reply by 2026-07-13 to confirm this is still relevant and reproducible on the current version of HAMi. If we do not hear back, we will close this issue. You can always reopen it or file a new one if the problem persists.

Thank you for contributing to HAMi.
